src/components/Navbar.tsx

```diff
@@ -57,20 +44,22 @@ const Header = () => {
               />
             </Link>
           </LinkScroll>
-        </a>
+
         <nav className="navbar__content ">
           <motion.ul initial={{ y: -250 }} animate={{ y: 0 }}>
-            <NavbarLink top="homePageTop" link="/" name="About" />
-            <NavbarLink top="teamPageTop" link="/team" name="Team" />
-            <NavbarLink
-              top="projectsPageTop"
-              link="/projects"
-              name="Projects"
-            />
-            <NavbarLink top="applyPageTop" link="/apply" name="Apply" />
-            <NavbarLink top="merchPageTop" link="/merch" name="Merch" />
-            <NavbarLink top="galleryPageTop" link="/gallery" name="Gallery" />
-            <NavbarLink top="docsPageTop" link="/resources" name="Resources" />
+            <li className="navbar__item">
+              <Link to="/" > About </Link>
+              <ul className="dropdown">
+                <li onClick={() => scrollToSection('projects')}><Link to="/" onClick={hideNavbar}>Our Projects</Link></li>
+                <li onClick={() => scrollToSection('events')}><Link to="/" onClick={hideNavbar}>Events</Link></li>
+                <li onClick={() => scrollToSection('sponsors')}><Link to="/" onClick={hideNavbar}>Sponsors</Link></li>
+              </ul>
+            </li>
+            <li><Link to="/team" onClick={hideNavbar}>Team</Link></li>
+            <li><Link to="/projects" onClick={hideNavbar}>Projects</Link></li>
+            <li><Link to="/apply" onClick={hideNavbar}>Apply</Link></li>
+            <li><Link to="/gallery" onClick={hideNavbar}>Gallery</Link></li>
+            <li><Link to="/resources" onClick={hideNavbar}>Resources</Link></li>
```

Comment:
Even though this is a simple component, it should still be a separate component IMO, since you're repeating all these `onClicks`.
I'd keep it more like the old version, and add a prop to control scrolling so we can use one component for all links

```javascript
<NavbarLink link="/team" name="Team" />
{/* Dropdown Link: */}
<NavbarLink link="/" scrollToSection="projects" />
```